docs/explanations/getting-started.md

```diff
@@ -5,34 +5,52 @@ This chapter will briefly introduce a few concepts that should help you learn QI
-The amplicon distribution, and other tools such as [MOSHPIT](https://moshpit.readthedocs.io/) (formerly referred to as the *metagenome distribution*) and [genome-sampler](https://genome-sampler.readthedocs.io/), are technically *built on top of the QIIME 2 Framework*.
+*QIIME 2* is built on what we called the QIIME 2 Framework (or *the framework*), and are in the process of rebranding as `rachis`.
+From here on, we'll refer to the QIIME 2 Framework as `rachis`.
+`rachis` is where the general-purpose functionality exists, including data provenance tracking, the plugin manager, and more.
+There is no microbiome-specific functionality (or even bioinformatics-specific functionality) in `rachis`.
+As an end user, you don't really need to know anything about this, but it's helpful to know that `rachis` exists and is different from QIIME 2 to understand the ecosystem of tools and how they can build on each other.
+QIIME 2, and other tools such as [MOSHPIT](https://moshpit.readthedocs.io/) (formerly referred to as the *QIIME 2 metagenome distribution*) and [genome-sampler](https://genome-sampler.readthedocs.io/), are technically *built on top of `rachis`*.
 
-**The amplicon distribution of QIIME 2 includes a suite of plugins that provide broad analytic functionality that supports microbiome marker gene analysis from raw sequencing data through publication quality visualizations and statistics.**
-There is not a single QIIME 2 workflow or command - rather it is a series of steps, and you choose which ones to apply.
-We provide general guidance through tutorials, like the [](#moving-pictures-tutorial), and can provide more specific guidance on the [QIIME 2 Forum](https://forum.qiime2.org).
-Any amplicon is supported - not just the 16S rRNA gene.
-The plugins that come with the amplicon distribution are listed in [](#available-plugins).
-Other plugins can also be installed independently - your main source for discovery and installation instructions for these is the [QIIME 2 Library](https://library.qiime2.org)[^developing-plugins].
+If you want to learn more about how `rachis` works, and how you can leverage it to become a QIIME 2 power user, you can refer to our book on that topic, [Using QIIME 2](https://use.qiime2.org)[^using-qiime2-rename].
+*Using QIIME 2* provides information that is relevant across all `rachis` distributions (QIIME 2, MOSHPIT) and stand-alone plugins.
+
+🐦‍⬛ 🪓 🦁
+:::
 
 ## Important concepts
 
 The following sections briefly present some important concepts for understanding QIIME 2 tools.
 You don't need to fully understand these to start using QIIME 2, but we think it will help you learn and build your bioinformatics skills if you have some brief exposure to these ideas.
-Links to where you can learn more are provided[^power-user].
+Links to where you can learn more are provided.
```

Comment:
```suggestion

```

Reply:
Hmm, I may have already made this change and forgot to push it up.